### DISCUSSION TITLE
Custom Extension to change html tag of Highlight

### DISCUSSION THREAD

**Discussion by jinny-0909** [ueberdosis/tiptap#5337] [General Questions] [answered]
Hello, 
I'm wondering if its possible to change the html tag being used for Highlight extension. The documentation says "You can use only default <mark> HTML tag, which has a yellow background color by default, or apply different colors." Seems like only mark could be used. 

But I wanted to check if we create a custom extension for Highlight and change html tag to <span> and set background-color to set highlight. The reason is that we are trying to create a text editor that drafts email content in the website, and view that email from email clients like Outlook. Noticed that Outlook uses <span> tag instead of <mark> for highlighting text. This resulted in when loading content received from Outlook into our TipTap editor, it doesn't render highlight correctly.

**Comment by nperez0111** [comment, marked as answer]
You can extend the highlight extension and overwrite the renderHTML method on it to be whichever tag you like. See our docs for extending existing extensions. And look at the code for highlight to see what it was doing before. It may also need the parseHTML method overridden if you change the input too.
